File: app/src/main/java/com/app/widgets/utils/fonts/Fonts.java
package com.app.widgets.utils.fonts;

import android.content.Context;
import android.graphics.Typeface;
import android.widget.TextView;

public class Fonts {

    // Font - paths
    private static String FONT_REGULAR = "fonts/" + "roboto_regular.ttf";

    public static Typeface getFontTypeface(Context context) {
        return Typeface.createFromAsset(context.getAssets(), FONT_REGULAR);
    }


}

File: app/src/main/java/com/app/widgets/utils/fonts/Fonts.kt
package com.app.widgets.utils.fonts

import android.content.Context
import android.graphics.Typeface

object Fonts {
    // Font - paths
    private const val FONT_REGULAR = "fonts/" + "roboto_bold.ttf"

    @JvmStatic
    fun getFontTypeface(context: Context): Typeface {
        return Typeface.createFromAsset(context.assets, FONT_REGULAR)
    }
}